app/controllers/api/scores_controller.rb

```diff
@@ -5,7 +5,9 @@ class ScoresController < ApplicationController
     before_action :validate_score_user_id, only: :destroy
 
     def user_feed
-      scores = Score.all.order(played_at: :desc, id: :desc)
+      # includes(:user) eagerly retrieves the user associated with the score
+      # this prevents hitting the database whenever the user associated with a certain score is needed
```

Comment:
It's good to have many comments when learning a new technology, but in products, we usually write comments related to the logic of our implementation, not built in functions.